client.go

```diff
@@ -118,3 +124,30 @@ func newClientUsingAddresses(port uint16, remoteAddresses []string) (*Client, er
 		return newClientUsingCluster(cluster)
 	}
 }
+
+// Get is a simple invocation of the NewFetchValueCommand
+func (c *Client) Get(opts *GetOptions) ([]*Object, error) {
+	builder := NewFetchValueCommandBuilder().
+		WithBucket(opts.Bucket).
+		WithKey(opts.Key)
+
+	if opts.BucketType != "" {
```

Comment:
There's no need for this check, `.WithBucketType` accepts a 0-length string and [the validation here](https://github.com/basho/riak-go-client/blob/master/locatable.go#L28-L30) will ensure the default bucket type is used in this case.